contrib/charts/cassandra/templates/cassandracluster.yaml

```diff
@@ -15,10 +15,10 @@ spec:
   - name: ringnodes
     replicas: {{ .Values.replicaCount }}
   image:
-    repository: {{ .Values.image.repository }}
-    tag: {{ .Values.image.tag }}
-    pullPolicy: {{ .Values.image.pullPolicy }}
+    repository: {{ .Values.image.repository | quote }}
+    tag: {{ .Values.image.tag | quote }}
```

Comment:
I got helm lint errors when I used an unquoted integer here.

https://github.com/kubernetes/helm/blob/master/docs/charts_tips_and_tricks.md#quote-strings-dont-quote-integers suggests quoting all strings, so I decided to do that.